governing-documents/project-annual-review.md

```diff
@@ -10,81 +10,89 @@ nav_order: 5
+    - change the `title` line to `YYY Annual Review Project Name` (e.g., `title: 2024 Annual Review Hyperledger Amazing`)
+    - change the `parent` line to `YYYY` (e.g., `parent: 2024`)
+    - change the `grand_parent` to `Project Updates` (i.e., `grand_parent: Project Updates`)
+    - remove the `nav_exclude` line
+  - Text between `<mark></mark>` are instructions. Please remove when section has been completed.
+- Send an email to the **LINK TBD** `TAC mailing list` so that the community knows the PR is there and can comment on it.
 
 ## Potential Outcomes
+
 The outcome of the annual review is:
 
-* A majority of the TAC members agree to continue the project at its current stage, or
-* A majority of the TAC members recommend that a project be moved to a new stage, including `End of Life`, or
-* If the TAC cannot come to a consensus, the project will remain at its current stage.
+- A majority of the TAC members agree to continue the project at its current status, or
+- A majority of the TAC members recommend that a project be moved to a new status, including `End of Life`, or
+- If the TAC cannot come to a consensus, the project will remain at its current status.
 
-NOTE: If the TAC members recommend moving to a new stage, additional work may be required to provide details on how the project meets the new stage's acceptance criteria.
+NOTE: If the TAC members recommend moving to a new status, additional work may be required to provide details on how the project meets the new status's acceptance criteria.
 
 ## Roles and Responsibilities
 
-### Hyperledger Staff
+### LF Decentralized Trust Staff
 
-- Maintaining the calendar for the project annual reviews
-- Notifying the project maintainers and copying the TAC when the project review is due
+- Maintaining the calendar for the project annual reviews.
+- Notifying the project TSC and copying the TAC when the project review is due.
 
-### Project Maintainers
-- Agreeing between themselves who will complete the annual review and informing the TAC of who will be completing the report
-- Creating the annual review by filing a PR in GitHub under [hyperledger/toc/project-reports/](https://github.com/hyperledger/toc/tree/gh-pages/project-reports)
-- Sending email to the [TAC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it
-- Optionally attending the public TAC meeting to engage in TAC-led discussion around the project. The project's maintainers are invited to use this time to voice any concerns and requests for help they may have that are not captured in the PR (or highlight asks within the PR)
+### Project TSC
+
+- Agreeing between themselves who will complete the annual review.
+- Informing the community about the annual review and requesting input.
+- Creating the annual review by filing a PR in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```

Comment:
Q: Is the github link a TODO?

Reply:
I think I just need to change `toc` to `tac` in it.  But a link check should be run on all of these docs after publication to make sure they are right.

Reply:
Changed the `toc` to `tac` where appropriate. There is one instance that was purposely left.